src/gfx_layout.cpp

```diff
@@ -671,7 +671,9 @@ Layouter::Layouter(const char *str, int maxw, TextColour colour, FontSize fontsi
 		} else {
 			/* Line is new, layout it */
 			FontState old_state = state;
+#if defined WITH_ICU_LAYOUT || defined WITH_UNISCRIBE
```

Comment:
I don't think we have any definite standard, but existing code seems to use the form with `()`.

Reply:
You are right. I'll change that.